### PR TITLE
avoid <<= operator, now deprecated in sbt

### PR DESCRIPTION
**Pull request by SethTisue**
smoothing the way to upgrading the build to sbt 0.13.13 when
it comes out

review by @dwijnand

### CODE REVIEW

**Comment by SethTisue** [comment]
test run https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/830/consoleFull (404 til Jenkins gets off the drugs and high on life)

File: community.dbuild [hunk 1]
```diff
@@ -44,7 +44,7 @@ set commands ++= {
     val allDefs = r._1s.toSeq
     val projectScope = Load.projectScope(currentRef)
     val scopes = allDefs.filter(_.key == scalacOptions.key).map(_.scope).distinct
-    val redefined = scopes.map(scope => scalacOptions in scope <<= (scalacOptions in scope).map(fn))
+    val redefined = scopes.map(scope => scalacOptions in scope ~= (_.map(fn)))
```
Comment (dwijnand) [inline community.dbuild]: no it needs to be something like `scalacOptions in scope ~= fn`

**Review by dwijnand — CHANGES_REQUESTED** [review]
Also, give it a quick 1-project run-through, just to check the macro doesn't throw a fit..

**Comment by SethTisue** [comment]
https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/832/consoleFull

**Review by dwijnand — APPROVED** [review]

**Comment by SethTisue** [comment]
the spire error looks unrelated to me

**Pull request merged**